### ISSUE TITLE
globalShortcut doesn't have keydown and keyup event

### ISSUE THREAD

**Issue by wuxiaoxiong** [electron/electron#26301]
### Problem Description
I want to register global shortcut and listen the shortcut `keydown` and `keyup` event when application does not have keyboard focus

Is there any module can do this?

It seems `globalShortcut` can't do this.

----

PS: 
I want to register  `F2` as shortcut. when user pressing `F2`, the app will open mic, and close mic after release `F2`

**Comment by ckerr** [comment]
Thanks for reaching out!

The issues tracker is used for feature requests and bug reports, so you will probably have better luck with this question at one of the links at [the project's Community page](https://www.electronjs.org/community) -- in particular, be sure to give [the Electron Discord server](https://discord.com/invite/electron) a try.

I hope this helps!
